utilities/transactions/transaction_test.cc

```diff
@@ -3132,6 +3128,17 @@ TEST_P(TransactionTest, LostUpdate) {
 }
 
 TEST_P(TransactionTest, UntrackedWrites) {
+  // In WriteUnprepared, untracked writes will break snapshot validation logic.
+  // Snapshot validation will only check the largest sequence number of a key to
+  // see if it was committed or not. However, an untracked unprepared write will
+  // hide smaller committed sequence numbers.
+  //
+  // TODO(lth): To fix this, snapshot validation will have to validate all
```

Comment:
I do not remember if we have documented this TODO somewhere else or not but if WrteUnprepared is imposing a new limitation it should be documented in the main code base, rather than in the test files. If we have such place in the main code base, the TODO might be better off there.

Reply:
I don't think there's a convenient place for me to put this, so I'll just implement my TODO.

Reply:
Put it in the header file